.pre-commit-config.yaml

```
@@ -27,12 +38,8 @@ repos:
           - --fuzzy-match-generates-todo
         types: [shell]
         exclude: ^internal/(?:go|pprof)/
-  - repo: https://github.com/golangci/golangci-lint
-    rev: v1.51.1
-    hooks:
-      - id: golangci-lint
```

I think overriding the `entry` of the existing hook should be enough and still ensure the correct version is available and used

```suggestion
  - repo: https://github.com/golangci/golangci-lint
    rev: v1.51.1
    hooks:
      - id: golangci-lint
        entry: make go/lint
```

---

I'll give this a try. Thanks for pointing that out ❤️